src/dsp_tools/resources/logger_config.yml

```
@@ -0,0 +1,19 @@
+handlers:
+  - sink: ext://sys.stdout
+    format: "{time:YYYY-MM:DD HH:mm:ss.SSSS} | <level>{level} | {message}</level>"
```

And for the others, accordingly:

```suggestion
    format: "{time:YYYY-MM-DD HH:mm:ss.SSS} | <level>{level} | {message}</level>"
```